plugins/wp/index.js

```diff
@@ -23,17 +23,24 @@ module.exports = function (app) {
     // Get latest 3 blog posts and pass it to home template
     let posts = await Model.getListOfPosts({
       number: 3,
-      fields: 'slug,title,content,date,modified,featured_image,categories'
+      fields: 'slug,title,content,date,modified,featured_image,categories,attachments'
     })
     posts = posts.map(post => {
+      const postAttachments = post.attachments ? Object.values(post.attachments) : []
+      const featuredImageAttachments = Object
```

Comment:
:smiley: just to be super picky here still `featuredImageAttachments` instead of `featuredImageAttachment` (should be singular)

Reply:
No, that's not picky, it should be that way, you're right. Done